src/browser-frontend/model/ui-page-model.js

```diff
@@ -22,6 +22,7 @@ const TAB_STATES = {
 const UIPageModel = Immutable.Record({
   tabState: TAB_STATES.INITIAL,
   locationInputBarValue: '',
+  tabOwner: '',
```

Comment:
It seems more logical to have this as domain state instead of UI state. A tab owner isn't (conceptually) something visual, I think. Please move this to domain-page-model.